app/views/lesson_completions/create.js.erb

```diff
@@ -1,2 +1,7 @@
-$('.lesson-button-group').html("<%= j render 'lessons/lesson_buttons', lesson: @lesson, course: @lesson.course, user: @user  %>");
 $('.banner__badge').html("<%= j render 'shared/course_badge', course: @lesson.course, user: @user, modifier: '' %>");
+
+<% if @came_from_lesson %>
```

Comment:
I think we should depend on the DOM to determine what to render here, something like 
`if $('.lesson-button-group').length > 0`

There might be a better way of doing this though, @arun1595 will surely know 😄